src/jit/compiler.h

```diff
@@ -5710,6 +5711,7 @@ class Compiler
                 __int64 lconVal;
                 double  dconVal;
                 Range   u2;
+                bool    edge;
```

Comment:
And speaking of `Equals` - it doesn't work properly in the case of `OAK_NO_THROW` assertions. It doesn't pay attention to `vnIdx` and `vnLen` and the only thing that keeps 2 different assertions from being considered equal is `O2K_INVALID` which always results in inequality. Of course, this means that we generate duplicate assertions. There are 2 large methods in corelib where we fail to eliminate a few range checks because the assertion table fills up.